### PR TITLE
Added some Text file

### PR DESCRIPTION
**Pull request by MohammdWakeel**
I have done some text file

### CODE REVIEW

File: wakeel.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hi every one
```
Comment (sargarakshay) [inline wakeel.txt:1]: Add your name and add your baranch in the file

Comment (MohammdWakeel) [reply wakeel.txt]: I have added branch name in the file

**Review by sargarakshay — APPROVED** [review]

**Review by sargarakshay — APPROVED** [review]
LGTM

**Review by lalitghongade — APPROVED** [review]
it looks good

**Review by sumansuruchi — APPROVED** [review]
All good

**Pull request merged**